src/createCli.js

```diff
@@ -32,41 +33,52 @@ const createCli = (argv) => {
     });
 
   // create a new app or lab
-  return cli
-    .command({
-      command: 'new',
-      desc: 'Create new Graasp app.',
-      builder: (_) => {
-        _.option('s', {
-          alias: 'starter',
-          type: 'string',
-          default: DEFAULT_STARTER,
-          describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
-        })
-          .option('f', {
-            alias: 'framework',
+  return (
+    cli
+      .command({
+        command: 'new',
+        desc: 'Create new Graasp app.',
+        builder: (_) => {
+          _.option('s', {
+            alias: 'starter',
             type: 'string',
-            describe: 'Set development framework (e.g. React, Angular)',
+            default: DEFAULT_STARTER,
+            describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
           })
-          .option('t', {
-            alias: 'type',
-            choices: ['app', 'lab'],
-            describe: 'Type of application (app or lab)',
-          })
-          .option('p', {
-            alias: 'path',
-            type: 'string',
-            describe: 'Path where project directory will be set up.',
-          });
-      },
-      handler: promisify(prompt),
-    })
-    .wrap(cli.terminalWidth())
-    .demandCommand(1, 'Pass --help to see all available commands and options.')
-    .strict()
-    .showHelpOnFail(true)
-    .recommendCommands()
-    .parse(argv.slice(2));
+            .option('f', {
+              alias: 'framework',
+              type: 'string',
+              describe: 'Set development framework (e.g. React, Angular)',
+            })
+            .option('t', {
+              alias: 'type',
+              choices: ['app', 'lab'],
+              describe: 'Type of application (app or lab)',
+            })
+            .option('p', {
+              alias: 'path',
+              type: 'string',
+              describe: 'Path where project directory will be set up.',
+            });
+        },
+        handler: promisify(prompt),
+      })
+      // package an app built files into a zip file
+      .command({
+        command: 'publish',
+        desc: 'Publish a zipped app',
```

Comment:
Add `.` for consistency with 'Create new Graasp app.' (or remove dot elsewhere).